galette/templates/default/pages/contribution_form.html.twig

```diff
@@ -75,10 +75,20 @@
                     <div class="field inline">
                         <label for="id_adh">{{ _T("Contributor:") }}</label>
                         <div id="id_adh" class="ui search selection dropdown nochosen">
+                    {% for k, v in members.list %}
```

Comment:
Unless I miss something, thee loop seems to be useless, a simple if/else should be enough